test/helpers/EnvTestFixture.hh

```diff
@@ -26,7 +26,8 @@
 using namespace ignition;
 
 /// \brief Common test setup for various tests
-class ServerFixture : public ::testing::TestWithParam<int>
+template <typename TestType>
+class InternalFixture : public TestType
```

Comment:
Would a better name for this class be `InternalEnvFixture`? I feel like the name `InternalFixture` doesn't really tell readers what the class does.

Reply:
My concern with the word "environment" is that in GoogleTest that's a separate concept. They have both `::testing::Test` and [::testing::Environment](https://github.com/YOU-i-Labs/googletest/blob/master/googletest/docs/V1_7_AdvancedGuide.md#global-set-up-and-tear-down), and what we're handling here is the test family, not the environments.

I'm open to other name alternatives though. I actually started with `GazeboFixture` :sweat_smile:

Reply:
> My concern with the word "environment" is that in GoogleTest that's a separate concept

Oh, I didn't realize that. I was just thinking that adding `environment` (or something similar) to the name could be good to imply to readers/users that this modifies environment variables. But, we can leave it named as-is.